rm-deployment-status-enhancer-tab/bower.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I would combine this with the previous sample ... it's more realistic and avoids a lot of duplication.